Benchmarks/Command.lean

```diff
@@ -12,58 +12,104 @@ initialize
     defValue := false
     descr := "enables/disables benchmarking in `withBenchmark` combinator"
   }
+  registerOption `benchmark.runs {
+    defValue := (5 : Nat)
+    descr := "controls how many runs the `benchmark` command does. \
+    NOTE: this value is ignored when the `profiler` option is set to true"
```

Comment:
By "this value" here, do you mean the default value of runs, or benchmarking altogether? I believe the former (and perhaps you run the benchmarks only once when the profiler is on?), but clarification would be nice.

Reply:
Indeed, I've made the description more explicit